### PR TITLE
Adding dropped Jet, Electron, Muon branches & Updating sample list for Run3

### PR DESCRIPTION
**Pull request by Ksavva1021**

### CODE REVIEW

File: NanoProd/crab/Run3_2022/Dihiggs.yaml [hunk 1]
```diff
@@ -51,7 +51,6 @@ GluGlutoRadiontoHHto2B2Tau_M-5000: /GluGlutoRadiontoHHto2B2Tau_M-5000_narrow_Tun
 
 GluGlutoHHto2B2Tau_kl-0p00_kt-1p00_c2-0p00: /GluGlutoHHto2B2Tau_kl-0p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
 GluGlutoHHto2B2Tau_kl-0p00_kt-1p00_c2-1p00: /GluGlutoHHto2B2Tau_kl-0p00_kt-1p00_c2-1p00_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
-GluGlutoHHto2B2Tau_kl-1p00_kt-1p00_c2-0p00: /GluGlutoHHto2B2Tau_kl-1p00_kt-1p00_c2-0p00_LHEweights_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
```
Comment (kandrosov) [inline NanoProd/crab/Run3_2022/Dihiggs.yaml:54]: these are two different samples, keep both and fix the "nanoprod" name. E.g. add suffix `_LHEweights`

File: NanoProd/crab/Run3_2023BPix/Dihiggs.yaml [hunk 2]
```diff
@@ -10,7 +10,6 @@ GluGlutoHHto2B2Tau_kl-1p00_kt-1p00_c2-0p10: /GluGlutoHHto2B2Tau_kl-1p00_kt-1p00_
 GluGlutoHHto2B2Tau_kl-1p00_kt-1p00_c2-0p35: /GluGlutoHHto2B2Tau_kl-1p00_kt-1p00_c2-0p35_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v6-v2/MINIAODSIM
 GluGlutoHHto2B2Tau_kl-1p00_kt-1p00_c2-3p00: /GluGlutoHHto2B2Tau_kl-1p00_kt-1p00_c2-3p00_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v6-v2/MINIAODSIM
 GluGlutoHHto2B2Tau_kl-1p00_kt-1p00_c2-m2p00: /GluGlutoHHto2B2Tau_kl-1p00_kt-1p00_c2-m2p00_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v6-v2/MINIAODSIM
-GluGlutoHHto2B2Tau_kl-2p45_kt-1p00_c2-0p00: /GluGlutoHHto2B2Tau_kl-2p45_kt-1p00_c2-0p00_LHEweights_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v6-v2/MINIAODSIM
```
Comment (kandrosov) [inline NanoProd/crab/Run3_2023BPix/Dihiggs.yaml:13]: same here: keep both samples

**Review by kandrosov — CHANGES_REQUESTED** [review]
Thank you  @Ksavva1021 ! Please add newly arrived samples listed below:
```
VBFHHto2B2Tau_CV_1_C2V_0_C3_1 is not available in: Run3_2023BPix, Run3_2023
  era=Run3_2022 file=NanoProd/crab/Run3_2022/Dihiggs.yaml dataset=/VBFHHto2B2Tau_CV_1_C2V_0_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
  era=Run3_2022EE file=NanoProd/crab/Run3_2022EE/Dihiggs.yaml dataset=/VBFHHto2B2Tau_CV_1_C2V_0_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
 Run3_2023BPix potential candidates from DAS:
    /VBFHHto2B2Tau_CV_1_C2V_0_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v6-v2/MINIAODSIM
 Run3_2023 potential candidates from DAS:
    /VBFHHto2B2Tau_CV_1_C2V_0_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer23MiniAODv4-130X_mcRun3_2023_realistic_v15-v2/MINIAODSIM
VBFHHto2B2Tau_CV_1_C2V_1_C3_1 is not available in: Run3_2023BPix, Run3_2023
  era=Run3_2022 file=NanoProd/crab/Run3_2022/Dihiggs.yaml dataset=/VBFHHto2B2Tau_CV_1_C2V_1_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
  era=Run3_2022EE file=NanoProd/crab/Run3_2022EE/Dihiggs.yaml dataset=/VBFHHto2B2Tau_CV_1_C2V_1_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
 Run3_2023BPix potential candidates from DAS:
    /VBFHHto2B2Tau_CV_1_C2V_1_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v6-v2/MINIAODSIM
 Run3_2023 potential candidates from DAS:
    /VBFHHto2B2Tau_CV_1_C2V_1_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer23MiniAODv4-130X_mcRun3_2023_realistic_v15-v2/MINIAODSIM
GluGlutoHHto2B2V_kl-1p00_kt-1p00_c2-0p00 is not available in: Run3_2023BPix, Run3_2023
  era=Run3_2022 file=NanoProd/crab/Run3_2022/Dihiggs.yaml dataset=/GluGlutoHHto2B2V_kl-1p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v3/MINIAODSIM
  era=Run3_2022EE file=NanoProd/crab/Run3_2022EE/Dihiggs.yaml dataset=/GluGlutoHHto2B2V_kl-1p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v3/MINIAODSIM
 Run3_2023BPix potential candidates from DAS:
    /GluGlutoHHto2B2V_kl-1p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v6-v2/MINIAODSIM
 Run3_2023 potential candidates from DAS:
    /GluGlutoHHto2B2V_kl-1p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer23MiniAODv4-130X_mcRun3_2023_realistic_v15-v1/MINIAODSIM
GluGlutoHHto2B2Vto2L2Nu_kl-1p00_kt-1p00_c2-0p00 is not available in: Run3_2023BPix, Run3_2023
  era=Run3_2022 file=NanoProd/crab/Run3_2022/Dihiggs.yaml dataset=/GluGlutoHHto2B2Vto2L2Nu_kl-1p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v1/MINIAODSIM
  era=Run3_2022EE file=NanoProd/crab/Run3_2022EE/Dihiggs.yaml dataset=/GluGlutoHHto2B2Vto2L2Nu_kl-1p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
 Run3_2023BPix potential candidates from DAS:
    /GluGlutoHHto2B2Vto2L2Nu_kl-1p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v6-v2/MINIAODSIM
 Run3_2023 potential candidates from DAS:
    /GluGlutoHHto2B2Vto2L2Nu_kl-1p00_kt-1p00_c2-0p00_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer23MiniAODv4-130X_mcRun3_2023_realistic_v15-v1/MINIAODSIM
VBFHHto2B2V_CV_1_C2V_0_C3_1 is not available in: Run3_2023BPix, Run3_2023
  era=Run3_2022 file=NanoProd/crab/Run3_2022/Dihiggs.yaml dataset=/VBFHHto2B2V_CV_1_C2V_0_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v3/MINIAODSIM
  era=Run3_2022EE file=NanoProd/crab/Run3_2022EE/Dihiggs.yaml dataset=/VBFHHto2B2V_CV_1_C2V_0_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v3/MINIAODSIM
 Run3_2023BPix potential candidates from DAS:
    /VBFHHto2B2V_CV_1_C2V_0_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v6-v2/MINIAODSIM
 Run3_2023 potential candidates from DAS:
    /VBFHHto2B2V_CV_1_C2V_0_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer23MiniAODv4-130X_mcRun3_2023_realistic_v15-v2/MINIAODSIM
VBFHHto2B2V_CV_1_C2V_1_C3_1 is not available in: Run3_2023BPix, Run3_2023
  era=Run3_2022 file=NanoProd/crab/Run3_2022/Dihiggs.yaml dataset=/VBFHHto2B2V_CV_1_C2V_1_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v3/MINIAODSIM
  era=Run3_2022EE file=NanoProd/crab/Run3_2022EE/Dihiggs.yaml dataset=/VBFHHto2B2V_CV_1_C2V_1_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v3/MINIAODSIM
 Run3_2023BPix potential candidates from DAS:
    /VBFHHto2B2V_CV_1_C2V_1_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v6-v2/MINIAODSIM
 Run3_2023 potential candidates from DAS:
    /VBFHHto2B2V_CV_1_C2V_1_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer23MiniAODv4-130X_mcRun3_2023_realistic_v15-v2/MINIAODSIM
VBFHHto2B2Vto2L2Nu_CV_1_C2V_0_C3_1 is not available in: Run3_2023BPix, Run3_2023
  era=Run3_2022 file=NanoProd/crab/Run3_2022/Dihiggs.yaml dataset=/VBFHHto2B2Vto2L2Nu_CV_1_C2V_0_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
  era=Run3_2022EE file=NanoProd/crab/Run3_2022EE/Dihiggs.yaml dataset=/VBFHHto2B2Vto2L2Nu_CV_1_C2V_0_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v1/MINIAODSIM
 Run3_2023BPix potential candidates from DAS:
    /VBFHHto2B2Vto2L2Nu_CV_1_C2V_0_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v6-v2/MINIAODSIM
 Run3_2023 potential candidates from DAS:
    /VBFHHto2B2Vto2L2Nu_CV_1_C2V_0_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer23MiniAODv4-130X_mcRun3_2023_realistic_v15-v2/MINIAODSIM
VBFHHto2B2Vto2L2Nu_CV_1_C2V_1_C3_1 is not available in: Run3_2023BPix, Run3_2023
  era=Run3_2022 file=NanoProd/crab/Run3_2022/Dihiggs.yaml dataset=/VBFHHto2B2Vto2L2Nu_CV_1_C2V_1_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
  era=Run3_2022EE file=NanoProd/crab/Run3_2022EE/Dihiggs.yaml dataset=/VBFHHto2B2Vto2L2Nu_CV_1_C2V_1_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v1/MINIAODSIM
 Run3_2023BPix potential candidates from DAS:
    /VBFHHto2B2Vto2L2Nu_CV_1_C2V_1_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v6-v2/MINIAODSIM
 Run3_2023 potential candidates from DAS:
    /VBFHHto2B2Vto2L2Nu_CV_1_C2V_1_C3_1_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer23MiniAODv4-130X_mcRun3_2023_realistic_v15-v2/MINIAODSIM
QCD_HT800to1000_MLM is not available in: Run3_2023
  era=Run3_2022 file=NanoProd/crab/Run3_2022/QCD.yaml dataset=/QCD-4Jets_HT-800to1000_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
  era=Run3_2022EE file=NanoProd/crab/Run3_2022EE/QCD.yaml dataset=/QCD-4Jets_HT-800to1000_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
  era=Run3_2023BPix file=NanoProd/crab/Run3_2023BPix/QCD.yaml dataset=/QCD-4Jets_HT-800to1000_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v2-v4/MINIAODSIM
 Run3_2023 potential candidates from DAS:
    /QCD-4Jets_HT-800to1000_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer23MiniAODv4-130X_mcRun3_2023_realistic_v14-v2/MINIAODSIM
QCD_HT1000to1200_MLM is not available in: Run3_2022EE, Run3_2023
  era=Run3_2022 file=NanoProd/crab/Run3_2022/QCD.yaml dataset=/QCD-4Jets_HT-1000to1200_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
  era=Run3_2023BPix file=NanoProd/crab/Run3_2023BPix/QCD.yaml dataset=/QCD-4Jets_HT-1000to1200_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v2-v4/MINIAODSIM
 Run3_2022EE potential candidates from DAS:
    /QCD-4Jets_HT-1000to1200_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
 Run3_2023 potential candidates from DAS:
    /QCD-4Jets_HT-1000to1200_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer23MiniAODv4-130X_mcRun3_2023_realistic_v14-v2/MINIAODSIM
VBFHHto2B2Tau_CV-m1p83_C2V-3p57_C3-m3p39 is not available in: Run3_2023BPix, Run3_2022, Run3_2023
  era=Run3_2022EE file=NanoProd/crab/Run3_2022EE/Dihiggs.yaml dataset=/VBFHHto2B2Tau_CV-m1p83_C2V-3p57_C3-m3p39_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
 Run3_2023BPix no candidates from DAS
 Run3_2022 potential candidates from DAS:
    /VBFHHto2B2Tau_CV-m1p83_C2V-3p57_C3-m3p39_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
 Run3_2023 no candidates from DAS
VBFHHto2B2V_CV-m1p83_C2V-3p57_C3-m3p39 is not available in: Run3_2023BPix, Run3_2022, Run3_2023
  era=Run3_2022EE file=NanoProd/crab/Run3_2022EE/Dihiggs.yaml dataset=/VBFHHto2B2V_CV-m1p83_C2V-3p57_C3-m3p39_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
 Run3_2023BPix no candidates from DAS
 Run3_2022 potential candidates from DAS:
    /VBFHHto2B2V_CV-m1p83_C2V-3p57_C3-m3p39_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
 Run3_2023 no candidates from DAS
VBFHHto2B2Vto2L2Nu_CV-m1p83_C2V-3p57_C3-m3p39 is not available in: Run3_2023BPix, Run3_2022, Run3_2023
  era=Run3_2022EE file=NanoProd/crab/Run3_2022EE/Dihiggs.yaml dataset=/VBFHHto2B2Vto2L2Nu_CV-m1p83_C2V-3p57_C3-m3p39_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
 Run3_2023BPix no candidates from DAS
 Run3_2022 potential candidates from DAS:
    /VBFHHto2B2Vto2L2Nu_CV-m1p83_C2V-3p57_C3-m3p39_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
 Run3_2023 no candidates from DAS
QCD_HT1000To1200_MLM is not available in: Run3_2022, Run3_2023BPix
  era=Run3_2022EE file=NanoProd/crab/Run3_2022EE/QCD.yaml dataset=/QCD-4Jets_HT-1000to1200_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
  era=Run3_2023 file=NanoProd/crab/Run3_2023/QCD.yaml dataset=/QCD-4Jets_HT-1000to1200_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer23MiniAODv4-130X_mcRun3_2023_realistic_v14-v2/MINIAODSIM
 Run3_2022 potential candidates from DAS:
    /QCD-4Jets_HT-1000to1200_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
 Run3_2023BPix potential candidates from DAS:
    /QCD-4Jets_HT-1000to1200_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v2-v4/MINIAODSIM
QCD_HT800to10000_MLM is not available in: Run3_2022EE, Run3_2022, Run3_2023BPix
  era=Run3_2023 file=NanoProd/crab/Run3_2023/QCD.yaml dataset=/QCD-4Jets_HT-800to1000_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer23MiniAODv4-130X_mcRun3_2023_realistic_v14-v2/MINIAODSIM
 Run3_2022EE potential candidates from DAS:
    /QCD-4Jets_HT-800to1000_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
 Run3_2022 potential candidates from DAS:
    /QCD-4Jets_HT-800to1000_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
 Run3_2023BPix potential candidates from DAS:
    /QCD-4Jets_HT-800to1000_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer23BPixMiniAODv4-130X_mcRun3_2023_realistic_postBPix_v2-v4/MINIAODSIM

```

**Comment by Ksavva1021** [comment]
@kandrosov Added the requested changes

**Review by kandrosov — APPROVED** [review]
All looks good now. Thank you @Ksavva1021 !

**Pull request merged**